src/internal/client/limit/limit.go

```diff
@@ -0,0 +1,65 @@
+// Package limit provides primitives to limit concurrency.
```

Comment:
I don't think this needs to be copied over. This package should probably be separate from the client package anyways.

Reply:
I’ll lift it up out of the client package, since it's not used in there. I do think that it should still be internal, so that nothing under the server package refers to it.